README.md

```diff
@@ -4,18 +4,18 @@
 
 [Documentation (development version)](https://www.pygmt.org/dev) | [Contact](https://forum.generic-mapping-tools.org) | [TryOnline](https://github.com/GenericMappingTools/try-gmt)
 
-[![Latest version on PyPI](https://img.shields.io/pypi/v/pygmt.svg?style=flat-square)](https://pypi.python.org/pypi/pygmt)
-[![Latest version on conda-forge](https://img.shields.io/conda/v/conda-forge/pygmt?style=flat-square)](https://anaconda.org/conda-forge/pygmt)
-[![GitHub Actions Tests status](https://github.com/GenericMappingTools/pygmt/workflows/Tests/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests.yaml)
-[![GitHub Actions GMT Dev Tests status](https://github.com/GenericMappingTools/pygmt/workflows/GMT%20Dev%20Tests/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
+[![Latest version on PyPI](https://img.shields.io/pypi/v/pygmt)](https://pypi.org/project/pygmt)
+[![Latest version on conda-forge](https://img.shields.io/conda/v/conda-forge/pygmt)](https://anaconda.org/conda-forge/pygmt)
+[![GitHub Actions Tests status](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests.yaml/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests.yaml)
+[![GitHub Actions GMT Dev Tests status](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml/badge.svg)](https://github.com/GenericMappingTools/pygmt/actions/workflows/ci_tests_dev.yaml)
 [![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff)
-[![Test coverage status](https://codecov.io/gh/GenericMappingTools/pygmt/branch/main/graph/badge.svg?token=78Fu4EWstx)](https://app.codecov.io/gh/GenericMappingTools/pygmt)
+[![Test coverage status](https://codecov.io/gh/GenericMappingTools/pygmt/graph/badge.svg?token=78Fu4EWstx)](https://app.codecov.io/gh/GenericMappingTools/pygmt)
 [![CodSpeed Performance Benchmarks](https://img.shields.io/endpoint?url=https://codspeed.io/badge.json)](https://codspeed.io/GenericMappingTools/pygmt)
-[![Compatible Python versions.](https://img.shields.io/pypi/pyversions/pygmt.svg?style=flat-square)](https://pypi.python.org/pypi/pygmt)
-[![Discourse forum](https://img.shields.io/discourse/status?label=forum&server=https%3A%2F%2Fforum.generic-mapping-tools.org%2F&style=flat-square)](https://forum.generic-mapping-tools.org)
+[![Compatible Python versions](https://img.shields.io/pypi/pyversions/pygmt)](https://pypi.org/project/pygmt)
```

Comment:
Seems that there is no specific link to the supported Python versions. Currently we have here again the link to the PyPi website for PyGMT.
Hm, just wondering if we can maybe link here to the newly added "Minimum Supported Versions" Technical Reference page?

```suggestion
[![Compatible Python versions](https://img.shields.io/pypi/pyversions/pygmt)](https://www.pygmt.org/dev/minversions.html)
```

Reply:
That's a good point. Currently, we use the badge from https://shields.io/badges/py-pi-python-version, which means the supported Python versions on PyPI. 

[![Compatible Python versions](https://img.shields.io/pypi/pyversions/pygmt)](https://pypi.org/project/pygmt)

So I feel linking to https://pypi.org/project/pygmt makes more sense than https://www.pygmt.org/dev/minversions.html.

I also found another badge at https://shields.io/badges/python-version-from-pep-621-toml. It can show the Python version provided in the `pyproject.toml` file, so it reflects the supported Python versions for the PyGMT dev. For this badge, I feel it makes more sense to linking it to https://www.pygmt.org/dev/minversions.html:

![Python Version from PEP 621 TOML](https://img.shields.io/python/required-version-toml?tomlFilePath=https%3A%2F%2Fraw.githubusercontent.com%2FGenericMappingTools%2Fpygmt%2Frefs%2Fheads%2Fmain%2Fpyproject.toml)

I prefer this badge. What do you think?

Reply:
I think I like this one, as it is shorter and linking to our version overview appears to me more helpful for users.

Reply:
Ping @michaelgrund and @weiji14 for comments on adopting the new badge.

![Supported Python versions](https://img.shields.io/python/required-version-toml?tomlFilePath=https%3A%2F%2Fraw.githubusercontent.com%2FGenericMappingTools%2Fpygmt%2Frefs%2Fheads%2Fmain%2Fpyproject.toml)

Reply:
I've made the change. Now the badges look like:

<img width="926" alt="image" src="https://github.com/user-attachments/assets/d516d2a2-ed45-452c-b1bb-d8ff3f0e15e2">